modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZonesUtil.java

```diff
@@ -84,6 +84,9 @@ public class DistributionZonesUtil {
     /** Key value for zones' global state revision. */
     private static final String DISTRIBUTION_ZONES_GLOBAL_STATE_REVISION = "distributionZones.globalState.revision";
 
+    /** Key value for the last handled logical topology by Distribution zone manager. */
+    private static final String DISTRIBUTION_ZONES_LAST_HANDLED_TOPOLOGY = "distributionZones.globalState.lastHandledTopology";
```

Comment:
Minor, but: there is some naming inconsistency, this prefix also has the `.globalState.` as previous, but the name is `DISTRIBUTION_ZONES_LAST_HANDLED_TOPOLOGY` instead of `DISTRIBUTION_ZONES_GLOBAL_STATE_LAST_HANDLED_TOPOLOGY`.

Maybe we can use a shorter _GLOBAL_ and kill two birds with one stone: have consistent, but not too long naming.

Anyway, up to you, not a big deal.